### PR TITLE
Neutron: Patient rerouting, remove instructions on rx review

### PR DESCRIPTION
**Pull request by jknapp25**
Add patient rerouting (#503)
Remove patient instructions on rx review (#511)

### CODE REVIEW

File: apps/patient/src/api/internal.ts [hunk 1]
```diff
@@ -0,0 +1,144 @@
...
+
+/**
+ * Mutations
+ */
+
+export const markOrderAsPickedUp = async (orderId: string) => {
+  try {
+    const response: { markOrderAsPickedUp: boolean } = await graphQLClient.request(
+      MARK_ORDER_AS_PICKED_UP,
+      { markOrderAsPickedUpId: orderId }
```
Comment (captDaylight) [inline apps/patient/src/api/internal.ts:72]: `markOrderAsPickedUpId` is a lil confusing, maybe just `orderId`?

Comment (jknapp25) [reply apps/patient/src/api/internal.ts:72]: yeah I agree, not sure why we originally did that, updating will require updating the services backend so I'll leave for next time i revisit that mutation

File: apps/patient/src/views/Main.tsx [hunk 2]
```diff
@@ -1,86 +1,70 @@
...
+const OrderContext = createContext(null);
+export const useOrderContext = () => useContext(OrderContext);
 
 export const Main = () => {
   const [searchParams] = useSearchParams();
   const orderId = searchParams.get('orderId');
   const token = searchParams.get('token');
 
+  if (token) {
+    setAuthHeader(token);
```
Comment (captDaylight) [inline apps/patient/src/views/Main.tsx:23]: This is a side effect, should be wrapping this in an effect ... or would be ideally be handled via context / provider

File: apps/patient/src/views/Main.tsx [hunk 3]
```diff
@@ -1,86 +1,70 @@
...
-  const [error, setError] = useState<string | undefined>(undefined);
 
   const navigate = useNavigate();
   const location = useLocation();
 
   if (location.pathname !== '/canceled') {
     if (!orderId || !token) {
       console.error('Missing orderId or token in search params');
-      navigate('/no-match');
+      navigate('/no-match', { replace: true });
```
Comment (captDaylight) [inline apps/patient/src/views/Main.tsx:34]: side effect, sholud be  in a useEffect

Comment (jknapp25) [reply apps/patient/src/views/Main.tsx]: fixed

File: apps/patient/src/views/Main.tsx (hunk 2 above)
Comment (jknapp25) [reply apps/patient/src/views/Main.tsx]: fixed

**Review by captDaylight — APPROVED** [review]

**Pull request merged**